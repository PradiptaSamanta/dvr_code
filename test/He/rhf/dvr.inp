dvr
rlimits  0.0 20.0 
mapped-grid 0.001
n-grids 200
n-gl 5
rot-qn-max 2
num-eig-value-fac 1.0
nuclear-charge 2
mass 1.0
dvr-diag
dvr-integrals
direct-integrals
debug 6
enddvr

rhf
maxiter 50
density-tol 1e-5
endrhf

orbital
num-n-qn 3
endorbital

end
