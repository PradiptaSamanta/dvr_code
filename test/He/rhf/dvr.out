 Reading the input from the file : dvr.inp
 **********
 Setting up these parameters:
 para%r_min     =     0.00
 para%r_max     =    20.00
 para%m         =       20
 para%nl        =        5
 ----------
 The radial grids are selected through mapping
 para%beta      =   0.001000
 para%E_max     =   0.000010
 ----------
 para%nr        =      101
 para%l_max     =        2
 para%nev       =       99
 para%Z         =        2
 para%mass      =     1.00
 ***********
 Initializing the GLL grid ...
    Original grid parameters were changed by differential map
               original                 after mapping
    r_min    0.00000000000000000E+00  0.00000000000000000E+00
    r_max    2.00000000000000000E+01  1.98612318968063262E+00
    nr          101                         101
 Using potential from file input_pot.in
   
 Grid initialization is done.
 Solving the radial Schroedinger equation for l =            0
 Time taken for diagonalization =     0.00800 seconds.
 Writing down the eigenvalues, eigenvectors and combining coefficients as demanded ...
 Solving the radial Schroedinger equation for l =            1
 Time taken for diagonalization =     0.01200 seconds.
 Writing down the eigenvalues, eigenvectors and combining coefficients as demanded ...
 Solving the radial Schroedinger equation for l =            2
 Time taken for diagonalization =     0.01200 seconds.
 Writing down the eigenvalues, eigenvectors and combining coefficients as demanded ...
 Calculating one-electron radial part of the integral for l =            0
 Calculating one-electron radial part of the integral for l =            1
 Calculating one-electron radial part of the integral for l =            2
 Calculating one-electron radial part of the integral for l =            3
 Calculating one-electron radial part of the integral for l =            4
 writing down the one-electron radial integrals
 two-electron integrals are calculated indirectly from an alternative expression
 Calculating two-electron radial part of the integral for l =            0
 Calculating two-electron radial part of the integral for l =            1
 Calculating two-electron radial part of the integral for l =            2
 Calculating two-electron radial part of the integral for l =            3
 Calculating two-electron radial part of the integral for l =            4
 writing down the two-electron radial integrals
 **********
 Setting up these parameters for the spherical harmonics:
 sph_harm%n_l         =        3
 sph_harm%n_mp        =        5
 ***********
 calculating the angular integrals
 Writing down the angular integrals
 Writing down the angular integrals
 ***** Doing RHF Calculation using FE-DVR basis *****
 Energy at the beginning:   -2.54485129980865     
 iter      error          Energy     time
   1     0.04458027    -2.55259313   0.12001
   2     0.00684085    -2.55274432   0.12001
   3     0.00100158    -2.55274752   0.12001
   4     0.00014587    -2.55274759   0.12001
   5     0.00002123    -2.55274759   0.11601
   6     0.00000309    -2.55274759   0.12001
 RHF calculation is converged in:           6 iterations...
 Final RHF energy:     -2.552747586592 Hartree.
 MO #  1energy=      -0.652029102817 occ=  2
 MO #  2energy=       0.784988132956 occ=  0
 MO #  3energy=       0.784988132956 occ=  0
 MO #  4energy=       0.784988132956 occ=  0
 MO #  5energy=       2.595263408971 occ=  0
 MO #  6energy=       2.595263408971 occ=  0
 MO #  7energy=       2.595263408971 occ=  0
 MO #  8energy=       2.595263408971 occ=  0
 MO #  9energy=       2.595263408971 occ=  0
 MO # 10energy=       2.745257996502 occ=  0
 MO # 11energy=       5.335057796897 occ=  0
 MO # 12energy=       5.335057796897 occ=  0
 MO # 13energy=       5.335057796897 occ=  0
 MO # 14energy=       8.483449343946 occ=  0
 MO # 15energy=       8.483449343946 occ=  0
 MO # 16energy=       8.483449343946 occ=  0
 MO # 17energy=       8.483449343946 occ=  0
 MO # 18energy=       8.483449343946 occ=  0
 MO # 19energy=       8.667451633079 occ=  0
 MO # 20energy=      12.527124891510 occ=  0
 ***** RHF is done...
 **********
 Setting up these parameters for the orbitals:
 orb%n_max     =        3
 orb%nSpatialOrbs     =       14
 ***********
 Time taken for 1e transformation =    0.00000 seconds.
 Time taken for 2e transformation =    0.11201 seconds.
 Time taken for combining integrals =    0.00400 seconds.
 Deallocated all the main matrices.
 The calculation is done. 
 Time =    1.20807500000000      seconds.
